test/base.py

```diff
@@ -1,12 +1,28 @@
 import unittest
-
 import jsonschema
+import os, sys
 
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from genson import Schema
 
 class SchemaTestCase(unittest.TestCase):
+
+    def assertGenSchema(self, instance, options, expected):
+        actual = Schema(**options).add_object(instance).to_dict()
+        self.assertSchema(actual, expected)
+        self.assertValidData(instance, actual)
```

Comment:
Ah, yes. We don't have access to the base object in that case. Yeah, this will have to stay separate. It's a good thing to test for though, and I see now why you created this method. There should be a way that we can abstract this into a method that we can also use it in `TestGenMulti` without sacrificing clarity. Let's think on it.